DIPs/DIP1012.md

```diff
@@ -53,9 +68,8 @@ As all the attributes are now symbols, we can:
 * group them in an `AliasSeq`-like fashion to apply them en masse, as is done in LDC for [`@fastmath`](https://github.com/ldc-developers/druntime/blob/ldc/src/ldc/attributes.d#L58)
 * mutate the `AliasSeq`s with compile-time computations.
 
-A module declaration may be tagged with zero or more attribute groups to apply to all symbols declared within the module and acting as the default. Templates are exceptioned and shall remain inferred with explicit tagging. Any attribute groups not provided will be inserted into the module's attribute list from the new `core.attribute.defaultAttributeSet`, the master set of atributes. 
-
-The values of `core.attribute.defaultAttributeSet` will be determined by version conditions under the control of compiler switches to provide maximum utility and flexibility, like so:
+The default value of the core attributes shall be taken from `core.attribute.defaultAttributeSet`.
+The values of `core.attribute.defaultAttributeSet` _may_ be determined by version conditions under the control of compiler switches to provide maximum utility and flexibility, like so:
```

Comment:
Yeah I don't expect that to be used very frequently and then only for applications. It is however very useful for ensuring all your dependancies are compatible with your attribute requirements, e.g. embedded, memory safety critical.